pkg/controllers/user/logging/utils/kafka.go

```diff
@@ -13,13 +14,19 @@ import (
 )
 
 var (
-	kafkaTestData = kafka.Message{Value: []byte(testMessage)}
+	defaultPartition = 0
```

Comment:
If we are using it as a constant, specify `const` for it, and the declaration in L183 is not needed.